packages/grid/docker-compose.yml

```diff
@@ -236,17 +236,15 @@ services:
       - blob-storage
     depends_on:
       - proxy
+    env_file:
+      - .env
     image: "${DOCKER_IMAGE_SEAWEEDFS?Variable not set}:${VERSION-latest}"
     environment:
       - S3_VOLUME_SIZE_MB=${S3_VOLUME_SIZE_MB:-1024}
       - S3_ROOT_USER=${S3_ROOT_USER:-admin}
       - S3_ROOT_PWD=${S3_ROOT_PWD:-admin}
       - S3_PORT=${S3_PORT:-8888}
       - SEAWEED_MOUNT_PORT=${SEAWEED_MOUNT_PORT:-4001}
```

Comment:
MAYBE: Since the seaweedfs dockerfile take the version as an argument,

```dockerfile
ARG SEAWEEDFS_VERSION

FROM chrislusf/seaweedfs:${SEAWEEDFS_VERSION}
```

 do we need to add it also the docker compose
 environment variables including SEAWEEDFS_VERSION

Reply:
yes we will have to update that as well.

Reply:
we pass the SEAWEEDFS_VERSION as an arg to docker compose build
![Uploading Screenshot from 2023-12-05 16-23-17.png…]()

Reply:
Yes we might to take care of devspace, that's true,

Reply:
added to devspace 
- [ad9f691](https://github.com/OpenMined/PySyft/pull/8301/commits/ad9f691b19dbe8b373e3277a8af924fc02edea9b)
- [1768a2d](https://github.com/OpenMined/PySyft/pull/8301/commits/1768a2d840b77b2499523e64f8eb08dc72d72f94)